src/test/java/org/fundacionjala/coding/Franco/AccountNumberTest.java

```diff
@@ -0,0 +1,93 @@
+package org.fundacionjala.coding.Franco;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Administrator on 8/25/2017.
+ */
+public class AccountNumberTest {
+    /**
+     * This method tests mapping of a number given
+     * in pipes and underscores to string of digits.
+     */
+    @Test
+    public void testMapEntryOCRToNumber() {
+        AccountNumber accountNumber = new AccountNumber();
+
+        List<String> lines1 = new ArrayList<>();
+        lines1.add("    _  _     _  _  _  _  _ ");
+        lines1.add("  | _| _||_||_ |_   ||_||_|");
+        lines1.add("  ||_  _|  | _||_|  ||_| _|");
+
+        final String actualResult1 = accountNumber.mapEntryOCRToNumber(lines1);
+        final String expectedResult1 = "123456789";
+        assertEquals(expectedResult1, actualResult1);
+
+        List<String> lines2 = new ArrayList<>();
+        lines2.add("    _  _     _  _  _  _  _ ");
+        lines2.add("  | _| _||_||_  _    |_||_|");
+        lines2.add("  ||_  _|  | _||_|  ||_| _|");
+
+        final String actualResult2 = accountNumber.mapEntryOCRToNumber(lines2);
+        final String expectedResult2 = "12345-1-189";
```

Comment:
if the number does not exist, it should be ? and not -
Change the implementation